examples/run_django_example.sh

```diff
@@ -0,0 +1,9 @@
+cd django-deployment
+make compile-packages
+make install
+brew install redis
+pipenv run python3 manage.py migrate
+pipenv run python manage.py createsuperuser
+FMA_SETTINGS_MODULE=federated_learning_project.fma_settings pipenv run python manage.py runserver &
+redis-server &
+FMA_SETTINGS_MODULE=federated_learning_project.fma_settings pipenv run python manage.py qcluster & fg
```

Comment:
We might rather want to separate combine from launch?

or instead of a separate script, have a `complete install` and ` launch all.